jsplot/src/main/java/tech/tablesaw/plotly/components/Marker.java

```diff
@@ -155,6 +163,39 @@ public String asJavascript() {
     return context;
   }
 
+  static class Color {
+
+    final int r, g, b;
+
+    public Color(int r, int g, int b) {
+      this.r = r;
+      this.g = g;
+      this.b = b;
+    }
+  }
+
+  private static String asColorScaleString(List<Double> values, List<Color> colors) {
+    StringBuilder sb = new StringBuilder();
+    sb.append("[\n");
+    for (int i = 0; i < values.size(); i++) {
+      Color c = colors.get(i);
+      if (i != 0) {
+        sb.append(",\n");
+      }
+      sb.append("['")
```

Comment:
If you add value to the color class as mentioned above you could factor this out into a separate method that takes no parameters.